src/theme/variables/edgeDark.ts

```diff
@@ -394,5 +397,8 @@ export const edgeDark: Theme = {
   guiPluginLogoMoonpay: guiPluginLogoMoonpay,
 
   // UI 4.0:
+  cardBackgroundUi4: palette.teal,
+  cardRadiusUi4: 1,
```

Comment:
We usually avoid putting dimensions in the theme, and instead rely on `rem` to scale things.

If we are committed to having the radius in the theme for some reason, then it's weird to do `theme.rem(theme.cardRadius)`. Maybe just return the radius in dp's, not rem's, since the theme controls the scaling already.

Reply:
Just checked, and the other theme dimensions are already in dp's, such as `thinLineWidth` or `secondaryButtonOutlineWidth`.

Reply:
This is a rem, I can rename the var

Reply:
I don't think it should be rem in the first place, but the rename is also an acceptable solution.